packages/embedder/indexing/embeddingsTablesOps.ts

```diff
@@ -52,7 +52,7 @@ export async function selectMetaToQueue(
     .where(({eb, not, or, and, exists, selectFrom}) =>
       and([
         or([
-          not(eb('em.models', '<@', sql`ARRAY[${sql.ref('model')}]::varchar[]` as any) as any),
+          not(eb('em.models', '@>', sql`ARRAY[${sql.ref('model')}]::varchar[]` as any) as any),
```

Comment:
There was a bug here!  We want to check if `em.models` is missing any configured models. Before this change, we were checking if any configured models were missing any elements in `em.models`. The only reason this produced queued jobs before was because `em.models` defaults to `NULL`.

The `<@` is the "is contained by" operator. It checks if the left-hand operand is contained within the right-hand operand. It returns true if the set on the right includes all elements or the specific structure on the left.

`ARRAY[2,3] <@ ARRAY[1,2,3]` returns true
`ARRAY[2] <@ ARRAY[1,2,3]` returns true  (not what we want!)

The `@>` operator  is the "contains" operator. It checks if the left-hand operand contains the right-hand operand. It returns true if the set on the left includes all elements or the specific structure on the right.

`ARRAY[1,2,3] @> ARRAY[2,3]` returns true
`ARRAY[1,2] @> ARRAY[2,3]` returns false (what we want) 

The better fix is to just stop using the models array...but we've already got that on our list. This just makes the current state work as designed.